cfme/tests/containers/test_pods_prop_data_integrity.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod
+from utils import testgen
+from utils.version import current_version
+from cfme.web_ui import CheckboxTable
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP-9934
+
+
+def test_summary_properties_validation(provider):
+    """
+          This test verifies that the number of running containers in the status summary table
+          is the same number that appears in the Relationships table
+    """
+    sel.force_navigate('containers_pods')
+    list_tbl_pod = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+    ui_pods = [r.name.text for r in list_tbl_pod.rows()]
+    ui_pods_revised = filter(
+        lambda ch: not ch.startswith('metrics'),
+        ui_pods)
+
+    for name in ui_pods_revised:
+        obj = Pod(name, provider)
```

Comment:
There is too much UI interaction going on here, this test should be something more like
```python
for pod in Provider.get_all_pods():
        # Perhaps need to check it's not the "metric" one in here, or make the get_all_pods take some kind of parameter to allow you to specificy
        val_cont_stat_summary = obj.get_detail(
            'Container Statuses Summary', 'Running')
        val_cont_rel_tbl = obj.get_detail('Relationships', 'Containers')
        assert val_cont_rel_tbl == val_cont_stat_summary
```

It's likely that ```get_all_pods``` as a function doesn't exist, but it should be done this way. It should also return Pod objects, not string names etc.